cli/src/pcluster/api/pcluster_api.py

```diff
@@ -386,9 +387,9 @@ def list_images(region: str):
 
             # get built images by image name tag
             images = AWSApi.instance().ec2.get_images()
-            imagebuilders = [ImageBuilder(image_name=image.original_image_name) for image in images]
+            imagebuilders = [ImageBuilder(image=image, image_name=image.original_image_name) for image in images]
             images_response = [
-                ImageBuilderImageInfo(imagebuilder=imagebuilder, image=imagebuilder.image)
+                ImageBuilderImageInfo(imagebuilder=imagebuilder)
```

Comment:
I still prefer passing image into ImageBuilderImageInfo, because ImageBuilderImageInfo just works as an information collecter, since we pass image into ImageBuilder, imagebuilder.image won't describe image again in this line.